src/Recordings.cpp

```diff
@@ -568,7 +568,14 @@ PVR_ERROR Recordings::SetRecordingLastPlayedPosition(const kodi::addon::PVRRecor
     time_t timerUpdate = m_timers.m_lastTimerUpdateTime;
     if (lastplayedposition == -1)
     {
-      lastplayedposition = recording.GetDuration();
+      if (recording.GetRecordingTime() + recording.GetDuration() > time(nullptr)) {
```

Comment:
`}` on the next line

Reply:
Sorry I've been doing a lot on Android Studio and Eclipse in the best month and all the Java and Kotlin braces are K&R style